### PR TITLE
README installation update

### PR DESCRIPTION
**Pull request by Davidslv**
I've just been through the installation of the app and I've came across some problems which I thought would be good to share on the README.

Thank you

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,14 +13,19 @@ user-friendly manner. Documentation can be found on [rdoc](http://rdoc.info/gith
...
 
-* Ruby 2.1.4
+* Xcode (for the Command Line Tools `xcode-select --install`)
+* Ruby 2.2.3
 * Rubygems and Bundler
 * Mysql
 * Imagemagick and Ghostscript (for generating thumbnails of uploaded
   PDFs)
-* xpdf
+* xpdf (first download [XQuarts](http://www.xquartz.org/))
```
Comment (boffbowsh) [inline README.md]: Nitpick, but it's `XQuartz` for the link text.

**Comment by boffbowsh** [comment]
This is great, but it's probably worth pointing out that you'll have a lot less friction if you run Whitehall from the developer VM.

File: README.md (hunk 1 above)
Comment (Davidslv) [inline README.md]: :+1: thanks

**Comment by boffbowsh** [comment]
:+1:

**Comment by jamiecobbett** [comment]
Hi @Davidslv, in case you didn't know, for Whitehall only, we merge our own pull requests, once they've been given a thumbs-up, so merge it when you're ready.

**Comment by Davidslv** [comment]
Ah! Thanks @jamiecobbett, I didn't know that
I will merge it then, thank you.

**Pull request merged**